Packs/CommunityCommonScripts/ReleaseNotes/1_4_1.md

```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+
+##### New: BulkCloseIncidents
```

Comment:
The heading for the new content item should not include the 'New: ' prefix. Please remove it to adhere to the required format.